src/ReleaseHistory.md

```diff
@@ -1,5 +1,10 @@
 # SARIF Package Release History (SDK, Driver, Converters, and Multitool)
 
+## Unreleased
+
+* BUGFIX: Fix issue `match-results-forward` command generates invalid Sarif when results contain sub ruleId. [#2486](https://github.com/microsoft/sarif-sdk/pull/2486)
```

Comment:
>BUGFIX: Fix issue `match-results-forward` [](http://example.com/codeflow?start=2&length=41)

I would either separate in two PRs so each PR has its own fix. Or, make an improvement in the text to be one entry that explains the current behavior for both issues and point to the correct PullRequest Id.

Reply:
Separated into 2 PRs.